setup.py

```diff
@@ -44,12 +44,14 @@
         'botocore==1.22.12',  # required by c7n 0.9.14
         'boto3==1.19.12',  # required by c7n 0.9.14
         'c7n==0.9.14',  # custodian
-        'requests==2.27.1',  # rest api & lambda
-        'PyGitHub==1.55',  # gitleaks
         'elasticsearch==7.10.0',  # depend on elasticsearch server
         "elasticsearch_dsl==7.4.0",
-        "sphinx==4.0.2",
-        "sphinx_rtd_theme==0.4.3",
+        'requests==2.27.1',  # rest api & lambda
+        'PyGitHub==1.55',  # gitleaks
+        "myst_parser==0.17.0",  # readthedocs
+        "sphinx==4.0.2",  # readthedocs
+        "sphinx_rtd_theme==0.4.3",  # readthedocs
+
     ],
```

Comment:
Please fix:
        "myst_parser==0.17.0",  # readthedocs => "myst-parser==0.17.0",
        "sphinx==4.0.2",  # readthedocs => "sphinx==4.5.0"
        "sphinx_rtd_theme==0.4.3",  # readthedocs => "sphinx-rtd-theme==1.0.0"

Reply:
Change " to '